controllers/core/openstackcontrolplane_controller.go

```diff
@@ -47,26 +47,6 @@ import (
 	k8s_errors "k8s.io/apimachinery/pkg/api/errors"
 )
 
-// GetClient -
-func (r *OpenStackControlPlaneReconciler) GetClient() client.Client {
-	return r.Client
-}
-
-// GetKClient -
-func (r *OpenStackControlPlaneReconciler) GetKClient() kubernetes.Interface {
-	return r.Kclient
-}
-
-// GetLogger -
-func (r *OpenStackControlPlaneReconciler) GetLogger() logr.Logger {
-	return r.Log
-}
-
-// GetScheme -
-func (r *OpenStackControlPlaneReconciler) GetScheme() *runtime.Scheme {
-	return r.Scheme
-}
```

Comment:
@stuggi Are these just essentially copy-pastes from our old pattern in OSPdO?  Just wondering if we still use this pattern somewhere in any of the new operators.

Reply:
yes, those are left overs from the older ReconcilerCommon and are now no longer needed with the newer Helper